### ISSUE TITLE
Data tracking for sidebar behavior learning

### ISSUE THREAD

**Issue by amywtlin** [forem/forem#14831]
https://docs.google.com/document/d/1ufKvwJosSv1fyAmm6cH27dCweiffM1f5IPlG2nPVSek/edit#

**Comment by amywtlin** [comment]
This is currently on hold. @JennieOcken will sort out overall data tracking policy with leadership first.

**Comment by mikeydorje** [comment]
Stale, blocked by tracking policy decisions at the time, and no movement since 2021. Closing due to age; reopen with updated goals + tracking policy alignment.
